$OpenBSD: patch-lib_ephy-profile-migrator_c,v 1.1 2017/02/06 17:57:47 jasper Exp $

ephy-profile-migrator.c:788:3: error: 'for' loop initial declarations are only allowed in C99 or C11 mode

--- lib/ephy-profile-migrator.c.orig	Mon Feb  6 18:42:12 2017
+++ lib/ephy-profile-migrator.c	Mon Feb  6 18:44:04 2017
@@ -785,7 +785,8 @@ migrate_insecure_passwords (void)
     goto out;
   }
 
-  for (GList *l = items; l != NULL; l = l->next)
+  GList *l;
+  for (l = items; l != NULL; l = l->next)
     migrate_insecure_password ((SecretItem *)l->data);
 
   g_list_free_full (items, g_object_unref);
